### PR TITLE
another fix.

### PR DESCRIPTION
**Pull request by sergey-shandar**

### CODE REVIEW

File: lib/util/utils.ts [hunk 1]
```diff
@@ -110,28 +127,15 @@ export async function parseJson(
         "https://raw.githubusercontent.com$2$3"
       )
     }
-    const res = makeRequest({ url: specPath, errorOnNon200Response: true })
+    const res = makeRequest({ url: specPath, errorOnNon200Response: true }).then(applySuppression)
```
Comment (billytrend) [inline lib/util/utils.ts]: why not async/await?

Comment (billytrend) [reply lib/util/utils.ts]: This is clearer imo

```
const swaggerObject = await makeRequest({ url: specPath, errorOnNon200Response: true })
docCache[specPath]  = applySuppression(res)
return docCache[specPath] 
```

Comment (billytrend) [reply lib/util/utils.ts]: or maybe i mean

```
const swaggerObject = await makeRequest({ url: specPath, errorOnNon200Response: true })
applySuppression(swaggerObject)
docCache[specPath]  = Promise.resolve(swaggerObject)
return swaggerObject
```

Comment (sergey-shandar) [reply lib/util/utils.ts]: We shouldn't await (block) for uploading document from HTTPS.

Comment (billytrend) [reply lib/util/utils.ts]: we do this already though... see line 132

**Comment by veronicagg** [comment]
@sergey-shandar could you add tests to cover the cases that weren't initially covered? and surfaced this bug?

**Pull request merged**